src/Raven.Server/Web/RequestHandler.cs

```diff
@@ -408,7 +409,8 @@ protected bool TryGetServerAdmin()
                 case RavenServer.AuthenticationStatus.Expired:
                 case RavenServer.AuthenticationStatus.Allowed:
                 case RavenServer.AuthenticationStatus.NotYetValid:
-                    if (Server.Configuration.Security.AuthenticationEnabled == false)
+                    if (Server.Configuration.Security.AuthenticationEnabled == false 
+                        && SecurityUtils.IsUnsecuredAccessAllowedForAddress(Server.Configuration.Security.UnsecuredAccessAddressRange, HttpContext.Connection.RemoteIpAddress))
```

Comment:
Wrong, we shouldn't be checking who is connecting to us. We should check the bind ip address and refuse to bind if this isn't valid.